falcon/request.py

```diff
@@ -272,10 +272,13 @@ def __init__(self, env, options=None):
         else:
             self.path = '/'
 
-        # PERF(kgriffs): if...in is faster than using env.get(...)
-        if 'QUERY_STRING' in env:
+        # # PERF(ueg1990): try/catch cheaper and faster (and more Pythonic)
```

Comment:
Sure! We've been using the new fancy "squash-and-merge" button for Github anyway, so if you don't squash, no worries. :-)